### PR TITLE
Update Command_report.java

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
Just added a feature so you cannot report a staff member.

### CODE REVIEW

**Comment by LegendIsAwesomes** [comment]
@Wild1145 I just tested on a quick 5 min server. It works perfectly fine.

File: src/com/superiornetworks/icarus/commands/Command_report.java [hunk 1]
```diff
@@ -72,6 +72,10 @@ public boolean run(CommandSender sender, Command cmnd, String string, String[] a
             {
                 admin.sendMessage(ChatUtils.colorize("&8[&4ICarusMod&8] &a" + sender.getName() + " &4Has reported " + Reported + " - " + player.getAddress().getAddress().getHostAddress() + " &4with the reason &2" + report_reason + "&4."));
             }
+            if (ICM_Rank.isRankOrHigher(player, ICM_Rank.Rank.SUPER))
+            {
+                sender.sendMessage(ChatColor.PURPLE + "Please do not try to report a staff member, If the staff member is abusing make sure to report him to a high enough rank to take action!");
```
Comment (Hockeyfan360) [inline src/com/superiornetworks/icarus/commands/Command_report.java]: Maybe "Please do not report staff members. If a staff member is abusing, take it to the forums."

It sounds better, if you know what I mean.

**Comment by LegendIsAwesomes** [comment]
@hockeyfan360 if this gets implemented put that instead of what I put.

Sent from phone.

**Comment by Wild1145** [comment]
@LegendIsAwesomes Any updates with this?

**Comment by Wild1145** [comment]
Looks good, just needs testing and then pulling in if all works as expected.

**Comment by Wild1145** [comment]
This does not compile. The chat colour you have used is invalid and as such it is impossible to compile this code.

**Comment by mgejay** [comment]
Its either DARK_PURPLE or LIGHT_PURPLE my friend.

**Comment by LegendIsAwesomes** [comment]
@Wild1145  https://github.com/Superior-Development/IcarusMod/pull/115/files#diff-4704d501f8288336432efcb326824c4e

@Scuph  thanks for the reminder

**Comment by Wild1145** [comment]
Will be tested shortly.

**Comment by Hockeyfan360** [comment]
You didn't return false in the command. In other words, this is useless. It basically sends them a chat message and does nothing else.

**Comment by LegendIsAwesomes** [comment]
@hockeyfan360 Sorry I abandoned this. I'll fix later.

**Comment by Hockeyfan360** [comment]
Please create a new pull request when you are ready to start on this again.

**Pull request closed without merging**